lua/bufferline/render.lua

```diff
@@ -344,7 +344,7 @@ end
 --- @param layout bufferline.layout.data
 local function open_buffer_start_animation(layout, bufnr)
   local buffer_data = state.get_buffer_data(bufnr)
-  local index = utils.index_of(state.buffers, bufnr)
+  local index = utils.index_of(Layout.buffers, bufnr)
 
   buffer_data.real_width = Layout.calculate_width(layout.base_widths[index], layout.padding_width)
```

Comment:
I'm getting this when opening a new tab with `:tabe` or `:tabe <new_file>`. Seems to work for `:tabe <already_open_file>`.

<img width="1792" alt="Screen Shot 2022-10-04 at 8 16 57 am" src="https://user-images.githubusercontent.com/7144420/193684859-ff8b21ae-3e87-40df-ae99-c10f5f10daa4.png">

```
Error detected while processing BufNew Autocommands for "*":
Error executing lua callback: .../pack/packer/start/barbar.nvim/lua/bufferline/layout.lua:147: attempt to perform arithmetic on local 'base_width' (a nil value)
stack traceback:
        .../pack/packer/start/barbar.nvim/lua/bufferline/layout.lua:147: in function 'calculate_width'
        .../pack/packer/start/barbar.nvim/lua/bufferline/render.lua:349: in function 'open_buffer_start_animation'
        .../pack/packer/start/barbar.nvim/lua/bufferline/render.lua:422: in function 'open_buffers'
        .../pack/packer/start/barbar.nvim/lua/bufferline/render.lua:576: in function 'get_updated_buffers'
        .../pack/packer/start/barbar.nvim/lua/bufferline/render.lua:946: in function 'update'
        .../pack/packer/start/barbar.nvim/lua/bufferline/render.lua:495: in function <.../pack/packer/start/barbar.nvim/lua/bufferline/render.lua:495>
```

Reply:
I can't seem to reproduce. What's your config? 

Also, have you updated to the most recent version of this branch? I rebased on `master` recently, so packer (and the like) may refuse to update unless you reinstall barbar or `git fetch --all; git reset --hard origin/feat/hide-buffers`

Reply:
I did uninstall and reinstall to get it, and I confirmed I am on `4d29318`. 

I cut down my config and was still able to reproduce with:

```lua
-- Bootstrap packer
local fn = vim.fn
local install_path = fn.stdpath("data") .. "/site/pack/packer/start/packer.nvim"
local repo_path = "https://github.com/wbthomason/packer.nvim"
if fn.empty(fn.glob(install_path)) > 0 then
  packer_bootstrap = fn.system({ "git", "clone", "--depth", "1", repo_path, install_path })
end
-- End bootstrap

-- Auto compile when there are changes in plugins.lua
local packer_user_config = vim.api.nvim_create_augroup("PackerUserConfig", {})
vim.api.nvim_create_autocmd("BufWritePost", {
  group = packer_user_config,
  pattern = "plugins.lua",
  command = "source <afile> | PackerCompile",
})

-- Limit parallel jobs to prevent failures
require("packer").init({
  max_jobs = 32,
})

return require("packer").startup(function(use)
  use("wbthomason/packer.nvim") -- Package manager
  use("kyazdani42/nvim-web-devicons")
  use({
    "romgrk/barbar.nvim",
    branch = "feat/hide-buffers",
    requires = "kyazdani42/nvim-web-devicons",
    config = function()
      require("bufferline").setup({
        hide = { inactive = true },
        auto_hide = true,
      })
    end,
  })

  -- Automatically set up configuration after cloning packer.nvim
  if packer_bootstrap then
    require("packer").sync()
  end
end)
```

Reply:
I was able to reproduce with the following command:

```sh
nvim --clean
     --cmd 'set rtp+=~/.local/share/nvim/site/pack/packer/start/barbar.nvim'
     --cmd 'set rtp+=~/.local/share/nvim/site/pack/packer/start/nvim-web-devicons'
     --cmd 'lua require"bufferline".setup {auto_hide = true, hide = {inactive = true}}'
```

Reply:
Latest update should fix it.

Reply:
Seems to work with no issues, thanks!